server/handlers.go

```diff
@@ -52,80 +57,507 @@ func handleRequestDownloaderMissing(logger *zap.Logger) http.HandlerFunc {
 	}
 }
 
-func handleAccountImages(multiaccountsDB *multiaccounts.Database, logger *zap.Logger) http.HandlerFunc {
-	return func(w http.ResponseWriter, r *http.Request) {
-		params := r.URL.Query()
+type ParsedParams struct {
```

Comment:
Perhaps rename this to `ImageParams` or `ProfileImageParams` or `PlaceHolderImageParams`